meilisearch-http/src/lib.rs

```diff
@@ -30,9 +29,9 @@ pub static AUTOBATCHING_ENABLED: AtomicBool = AtomicBool::new(false);
 pub fn setup_meilisearch(opt: &Opt) -> anyhow::Result<MeiliSearch> {
     let mut meilisearch = MeiliSearch::builder();
 
-    // enable autobatching?
+    // disable autobatching?
     AUTOBATCHING_ENABLED.store(
-        opt.scheduler_options.enable_auto_batching,
+        opt.scheduler_options.disable_auto_batching,
         std::sync::atomic::Ordering::Relaxed,
     );
```

Comment:
Are we sure about the names and booleans values, here. We changed the boolean but not the name of the global.

Reply:
Yup something where wrong in the display of tasks